CHANGES.md

```diff
@@ -1,5 +1,7 @@
 # Unreleased
 
+- [NEW] Moved `create_query_index` to `CouchDatabase` as it is now implemented there.
```

Comment:
Two things:
i) There are a bunch of other index/query related methods that moved as well.
ii) I know you are referring to new versions of CouchDB, but a newbie might not in which case "implemented there" is a bit confusing because it could mean `CouchDatabase`.

How about:
```
- [NEW] Moved `create_query_index` and other query related methods to `CouchDatabase` as the `_index`/`_find` API is available in CouchDB 2.x.
```

Reply:
Done :white_check_mark: